README.md

```diff
@@ -1,100 +1,132 @@
-# EasyEngine/cron-command
+easyengine/cron-command
```

Comment:
```# EasyEngine/cron-command```

It should be this syntax

Reply:
This comes from `package_name` defined in composer.json file.